packages/zowe-explorer/src/configuration/Profiles.ts

```diff
@@ -152,6 +152,36 @@ export class Profiles extends ProfilesCache {
             this.profilesForValidation.push(profileStatus);
         }
 
+        if (node !== undefined) {
+            const toolTipList = (node.tooltip as string).split("\n");
+
+            const autoStoreValue = (await this.getProfileInfo()).getTeamConfig().properties.autoStore;
+            const autoStoreIndex = toolTipList.findIndex((key) => key.startsWith(vscode.l10n.t("Auto Store: ")));
+            if (autoStoreIndex === -1) {
+                toolTipList.push(`${vscode.l10n.t("Auto Store: ")}${autoStoreValue.toString()}`);
+            } else {
+                toolTipList[autoStoreIndex] = `${vscode.l10n.t("Auto Store: ")}${autoStoreValue.toString()}`;
+            }
+
+            const layers = await this.uniqueExistingLayers();
+            const configFileIndex = toolTipList.findIndex((key) => key.startsWith(vscode.l10n.t("Config File: ")));
+            if (configFileIndex === -1) {
+                toolTipList.push(`${vscode.l10n.t("Config File: ")}${layers[0].global ? "Global" : "Project"}`);
+            } else {
+                toolTipList[configFileIndex] = `${vscode.l10n.t("Config File: ")}${layers[0].global ? "Global" : "Project"}`;
```

Comment:
Sorry to be picky with localizing strings... but the "Global" and "Project" strings should also be wrapped in `vscode.l10n.t`